webpack/assets/javascripts/react_app/actions/ServerActions.js

```diff
@@ -17,5 +17,20 @@ export default {
         errorThrown: errorThrown
       }
     });
+  },
+    receivedHostsPowerState(response, textStatus, jqXHR) {
+    AppDispatcher.dispatch({
+      actionType: ACTIONS.RECEIVED_HOSTS_POWER_STATE,
+      response
+    });
+  },
+  hostsRequestError(jqXHR, textStatus, errorThrown) {
+    AppDispatcher.dispatch({
+      actionType: ACTIONS.HOSTS_REQUEST_ERROR, info: {
+        jqXHR: jqXHR,
+        textStatus: textStatus,
+        errorThrown: errorThrown
+      }
+    });
```

Comment:
again, feels like boilerplate - the only change AFAICT is in the actionType parameter. Why not use the URL (or some other parameter) to reduce duplication?